src/nav/nav.ts

```diff
@@ -353,16 +353,16 @@ export class NgbNavLink {
  *
  * @since 5.2.0
  */
-export interface NgbNavChangeEvent {
+export interface NgbNavChangeEvent<T> {
```

Comment:
```suggestion
export interface NgbNavChangeEvent<T = any> {
```
This would work, otherwise the change will break the existing stuff (see CI failure)
With the default value for generic, the change is no brainer and useful and people don't have to pay attention to generics

Reply:
right!